semver.md

```diff
@@ -31,7 +31,7 @@ specified too loosely, you will inevitably be bitten by version promiscuity
 Dependency hell is where you are when version lock and/or version promiscuity
 prevent you from easily and safely moving your project forward.
 
-As a solution to this problem, we propose a simple set of rules and
+SemVer addresses this with a simple set of rules and
 requirements that dictate how version numbers are assigned and incremented.
 These rules are based on but not necessarily limited to pre-existing
```

Comment:
I think there should be a comma here, while you're at it.  Should be "...based on, but..."